### PR TITLE
fix: use gd token address from deployment.json

### PR DESCRIPTION
**Pull request by L03TJ3**
# Description
Replace hardcoded values
- [x] - use gd token address from deployment.json

About # (link your issue here)
#52

### CODE REVIEW

File: packages/common/src/constants/addresses.ts [hunk 1]
```diff
@@ -1,3 +1,2 @@
 // EXTERNAL CONTRACTS
-
-export const DAI_ADDRESS = '0x6B175474E89094C44Da98b954EedeAC495271d0F';
+export const DAI_ADDRESS = '0x6B175474E89094C44Da98b954EedeAC495271d0F'; // dai-mainnet
```
Comment (L03TJ3) [inline packages/common/src/constants/addresses.ts:2]: @sirpy 
there are some places where we define token addresess hardcoded.

Should these be pulled from somewhere, or are the WBTC/DAI type addreses pretty solid for now?

Comment (krisbitney) [reply packages/common/src/constants/addresses.ts:2]: There is quite obviously a difference between using a constant value and using mock data.

Comment (sirpy) [reply packages/common/src/constants/addresses.ts:2]: token addresses can be assumed to be constants
it says mainnet, this is a celo dapp. 
lets make sure they are correct
@krisbitney @L03TJ3

Comment (krisbitney) [reply packages/common/src/constants/addresses.ts:2]: This is in `packages/common`. It is not used in the app.

Comment (L03TJ3) [reply packages/common/src/constants/addresses.ts:2]: Ok, was just verifying. Then the answer to the orginal question is yes: all todo's are managed

**Pull request merged**